### DISCUSSION TITLE
Recovery Account

### DISCUSSION THREAD

**Discussion by AzhariBastomi** [community/community#134247] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My account Apoaqpeduli cannot be accessed because I forgot the 2-key, and the email has been unlinked. Is it possible to recover it?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Ap0AqPedul1 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
